docs/preview/features/powershell/azure-logic-apps.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Scripts related to interacting with Azure Logic Apps"
+layout: default
+---
+
+# Azure Logic Apps
+
+This module provides the following capabilities:
+- [Disabling Azure Logic Apps from configuration file](#disabling-azure-logic-apps-from-configuration-file)
+
+## Installation
+
+To have access to the following features, you have to import the module:
+
+```powershell
+PS> Import-Module -Name Arcus.Scripting.LogicApps
+```
+
+## Disabling Azure Logic Apps from configuration file
+
+Typically done the first task of the release pipeline, right before the deployment of the Logic Apps, will disable all specified Logic Apps in a specific order. 
+The Logic Apps to be disabled and the order in which this will be done, will be defined in the passed-along configuration file.
+
+| Parameter         | Mandatory | Description                                                                                                        |
+| ----------------- | --------- | ------------------------------------------------------------------------------------------------------------------ |
+| DeployFileName    | yes       | If your solution consists of multiple interfaces, you can specify the flow-specific name of the orderControl-file. |
+| ResourceGroupName | yes       | The resource group containing the Azure Logic Apps.                                                                |
+
+The schema of this configuration file is a JSON structure of an array with the following inputs:
+
+| Node        | Type            | Description                                                                                                           |
+| ----------- | --------------- | --------------------------------------------------------------------------------------------------------------------- |
+| Description | `string`        | Description of Azure Logic App set to disable.                                                                        |
+| CheckType   | `enum`          | Either `None` or `NoWaitingOrRunningRuns` which waits until there are no more waiting or running Logic App instances. |
+| StopType    | `enum`          | Either `None` or `Immediate` which results in the disabling of the Logic App.                                         |
+| LogicApps   | `string array`  | Set of Logic App names to disable.                                                                                    |
+
+**Example**
+
+With a 3 Azure Logic App (`"rcv-sthp-harvest-order-af-*"`), that doesn't looks for still-running Logic Apps (`checkType = None`) and t disables them (`stopType = Immediate`).
```

Comment:
I would change the provided sample/names to reflect something more specific (as this points to a specific existing customer setup)
```suggestion
Taking an example in which a set of Azure Logic Apps (`"rcv-shopping-order-*"`) need to be disabled, the following configuration will not take into account any active Logic Apps runs (`checkType = None`) and will immediately disable them (`stopType = Immediate`), starting with the _receive protocol_ instances and working its way up to the _sender_ Logic App.
```